packages/app-project/helpers/404theme.js

```diff
@@ -0,0 +1,65 @@
+const theme = {
+  heading: {
+    level: {
+      1: {
+        small: {
+          size: '37.5px',
+        },
+        medium: {
+          size: '48px',
+        },
+        large: {
+          size: '80px',
+        },
+        xlarge: {
+          size: '100px',
+        }
+      },
+      2: {
```

Comment:
Is there a Heading level=2 used in the 404 pages?

Reply:
No - I implemented the full set of styles that are possible to create from the Figma file.